coral-schema/src/main/java/com/linkedin/coral/schema/avro/RelToAvroSchemaConverter.java

```diff
@@ -232,6 +232,7 @@ public RelNode visit(LogicalProject logicalProject) {
 
     @Override
     public RelNode visit(LogicalJoin logicalJoin) {
+      // TODO: Modify this method to avoid that 2 schema fields share the same name in Avro 1.10, and enable the tests for join
```

Comment:
1. does this scenario mean using the same fieldNames, but referring to different columns, with different data types, across 2 schemas?
2. is this a non-blocking to-do? I see that some UTs have been disabled. Will there be a regression in production views if we don't implement it in this PR?

Reply:
1. Yes, and the data types may also be same
2. It's a non-blocking to-do, there is no regression on all the production views.